src/lib/style/theme.js

```diff
@@ -147,6 +147,51 @@ export const fontSize = {
   massive: '1.71rem', // no use case for the moment
 };
 
+export const space = [
+  '0',
+  '0.071rem',
+  '0.143rem',
+  '0.286rem',
+  '0.571rem',
+  '0.71rem',
+  '0.857rem',
+  '1rem',
+  '1.143rem',
+  '1.429rem',
+  '1.714rem',
+  '2rem',
+  '2.286rem',
+  '2.857rem',
+];
```

Comment:
We may want to base the old spacing on the items of this table to avoid duplicate mesurements